checker/votour.ml

```diff
@@ -116,6 +117,7 @@ struct
   | Ptr p ->
     match LargeArray.get !memory p with
     | Struct (tag, os) -> BLOCK (tag, os)
+    | Int63 _ -> OTHER (* FIXME *)
```

Comment:
This is fine, but you can also enhance the patch by adding a custom tag with hand-defined printers.